src/stores/businessStore.ts

```diff
@@ -90,15 +90,15 @@ export const useBusinessStore = defineStore('business', {
       return state.businessInfo.lastDirectorChangeDate
     },
 
-    /** The legal name, or operating name if is firm. */
+    /** The legal name, or alternate name if is firm. */
     getLegalName (state: BusinessStateIF): string {
-      return (this.isFirm && this.getOperatingName) ? this.getOperatingName : state.businessInfo.legalName
+      return (this.isFirm && this.getAlternateName) ? this.getAlternateName : state.businessInfo.legalName
     },
 
-    getOperatingName (state: BusinessStateIF): string {
+    getAlternateName (state: BusinessStateIF): string {
```

Comment:
Since we moved from `operatingName` to just `name`, I could've named this `getName` but I felt like that was very generic and might be misleading. So, I opted for `getAlternateName` instead.

Reply:
I like it. It's explicit (like legalName).